article/app/views/package.scala

```diff
@@ -84,7 +85,7 @@ object BodyProcessor {
         showAffiliateLinks = article.content.fields.showAffiliateLinks,
         contentType = "article",
         tags = article.content.tags.tags.map(_.id),
-        publishedDate = article.content.fields.firstPublicationDate,
+        isInDisableAffiliateLinksTest = ActiveExperiments.isParticipating(DisableAffiliateLinks)
```

Comment:
Can we move this check into the cleaner itself? It doesn't feel like it ought to be part of the API. It keeps the check closer to the code that uses it, which makes it easier to understand and clean up.